### ISSUE TITLE
[FileUpload]: WCAG React issues

### ISSUE THREAD

**Issue by tobre6** [TEHIK-EE/tedi-design-system#1020]
## Problem
9.1.3.1 - Ei loeta ette midagi lisanduvat, kui on tegemist üleslaadimisel ebaõnnestunud failiga. Peab seda tegema, sest muidu ei ole võimalik ekraanilugeri abil eristada õnnestunud ja ebaõnnestunud faile.
9.1.4.4 - 200% suurenduse juures “Read Only Files“ alad muutuvad kitsaks ja tekstid on aladest väljas. Sisu ise on loetav.
9.1.4.10 - Sulgemise nupuga fileupload’idel on “Add attachment“ nupp alast väljas. Ka on liiga suure laiusega failid alast väljas, näiteks “Multiple With Individual Validation“ all veamärgisega “taotlus_scan_lk_5.pdf.pdf“.
9.4.1.2 - Loeb faili lisamise nupu nime juurde tagasiside või abiteksti juhul, kui see on olemas. Siis ütleb nupu rolli ja siis loeb õiges kohas abiteksti. Ei tohi nupu nime juures lugeda abitekste.
9.4.1.3 - Faili lisamisel ei teavita edukast faililisamisest.
